KeeTrayTOTP/FormAbout.cs

```diff
@@ -156,5 +159,46 @@ private void FormAbout_FormClosed(object sender, FormClosedEventArgs e)
         {
             GlobalWindowManager.RemoveWindow(this);
         }
+
+        /// <summary>
+        /// Windows Form Mouse Click Handler.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListViewAbout_MouseClick(object sender, MouseEventArgs e)
+        {
+            var hit = ListViewAbout.HitTest(e.Location);
+            if (hit.Item != null && hit.Item == ListViewAbout.Items[4] && hit.SubItem == ListViewAbout.Items[4].SubItems[1])
+            {
+                var url = new Uri(hit.SubItem.Text); 
+                System.Diagnostics.Process.Start(url.ToString());
+            }
+        }
+        /// <summary>
+        /// Windows Form Row Subitem To Underlined Font.
+        /// </summary>
+        /// <param name="listView"></param>
+        /// <param name="RowIndex"></param>
+        /// /// <param name="color"></param>
+        private void ChangeToUnderlined(ref ListView listView, int RowIndex, Color color)
+        {
+            listView.Items[RowIndex].UseItemStyleForSubItems = false;
+            listView.Items[RowIndex].SubItems[1].Font = new Font("Microsoft Sans Serif", 8, FontStyle.Underline);
+            listView.Items[RowIndex].SubItems[1].ForeColor = color;
+
+        }
+
+        /// <summary>
+        /// Windows Form Mouse Move Handler.
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListViewAbout_MouseMove(object sender, MouseEventArgs e)
+        {
+            var hit = ListViewAbout.HitTest(e.Location);
+            if (hit.Item != ListViewAbout.Items[4]) ListViewAbout.Cursor = Cursors.Default;
```

Comment:
This can be simplified by integrating the first ``if`` reversed into the second one.

And please also put single-line statements in brackets. This makes it a little clearer.